packages/venia-ui/lib/components/ProductImageCarousel/carousel.css

```diff
@@ -45,6 +45,18 @@
     margin-right: 1.625rem;
 }
 
+.chevron {
+    composes: root from '../Icon/icon.css';
+    align-self: center;
+    border: 2px solid transparent;
+}
+
+.nextButton:focus > .chevron,
+.previousButton:focus > .chevron {
+    box-shadow: 0 0 4px 2px rgb(var(--venia-global-color-teal));
```

Comment:
CSS tokens don't exist on the release branch. I'm fine leaving this as is though, it'll just start working on `develop` and in the next branch.

Reply:
Probably worth changing now.

Reply:
Updated to use `--venia-teal` good catch @sirugh